setup.py

```diff
@@ -25,6 +25,7 @@
     "pendulum>=2.0.5,<3.0.0",
```

Comment:
We should also have the dependencies for the various DB possibilities as optional installs. See https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies